lib/libmbedtls/mbedtls/include/mbedtls/bignum.h

```diff
@@ -199,6 +206,18 @@ extern void *mbedtls_mpi_mempool;
 void mbedtls_mpi_init( mbedtls_mpi *X );
 void mbedtls_mpi_init_mempool( mbedtls_mpi *X );
 
+/**
+ * \brief           Initialize one MPI with supplied value. The size of
+ *		    the MPI is limited to max @alloc_size.
```

Comment:
Prefer spaces over tabs.
Here and in `bignum.c`.

Reply:
This commit will be removed since it's not needed any longer with the commit "libutee: lessen dependency on mbedtls internals"